Cargo.toml

```diff
@@ -15,6 +15,9 @@ maintenance = { status = "actively-developed" }
 
 [features]
 default = ["std"]
+rkyv_size_16 = ["rkyv/size_16"]
+rkyv_size_32 = ["rkyv/size_32"]
+rkyv_size_64 = ["rkyv/size_64"]
```

Comment:
These features are meant to be controlled by a top-level crate. I think that whatever uses glam and rkyv can choose these features and they'll unify all the way down.